CHANGELOG.rst

```diff
@@ -17,6 +17,8 @@ Changed
 -------
 - changed removing punctuation logic in ``WhitespaceTokenizer``
 - created a common utils package ``rasa.utils`` for nlu and core, common methods like ``read_yaml`` moved there
+- removed ``--num_threads`` from run command (server will be asyncronous but
```

Comment:
should we use the parameter for the number of sanic workers?

Reply:
I'm not sure what the design decision was behind this change. Maybe @tmbo can explain.

Reply:
@tmbo

Reply:
no I think we shouldn't add that at this point, as there is no multi-process implementation for ensuring user message ordering yet (making sure if a user sends two messages quickly after one another, they get handled in order and not in parallel)